packages/plugin/auto-unpack-natives/src/AutoUnpackNativesPlugin.ts

```diff
@@ -23,7 +23,7 @@ export default class AutoUnpackNativesPlugin extends PluginBase<AutoUnpackNative
       forgeConfig.packagerConfig.asar = {};
     }
     const existingUnpack = forgeConfig.packagerConfig.asar.unpack;
-    const newUnpack = '**/*.node';
+    const newUnpack = '**/{.webpack,**}/**/*.node';
```

Comment:
Is there a way to solve it that makes it work for all hidden directories, not just `.webpack`?

Reply:
yes, `**/{.**,**}/**/*.node` will do that. I'll fix it!